Lesson 1. Added products app and static files

AlexKovyazin opened the pull request.

Commit:
Lesson 1. Added products app and static files

vprinz commented:
1. У вас в settings.py нет в INSTALLED_APPS приложения `products` - из-за этого может не работать
2. Так же не подключена статика `STATICFILES_DIRS`
3. geekshop/urls.py - там нет адресов на контроллеры

Commit:
Lesson 1. Change settings.py and urls.py
settings.py - added STATICFILES_DIRS, added products to INSTALLED_APPS
urls.py - added paths to index and product controllers

AlexKovyazin commented:
> 1. У вас в settings.py нет в INSTALLED_APPS приложения `products` - из-за этого может не работать
> 2. Так же не подключена статика `STATICFILES_DIRS`
> 3. geekshop/urls.py - там нет адресов на контроллеры

По какой-то причине запушил до того, как внёс изменения в перечисленные файлы. Исправлено
